src/Http/Cookie/Cookie.php

```diff
@@ -535,6 +624,26 @@ public function isExpanded(): bool
         return $this->isExpanded;
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function getOptions(): array
```

Comment:
How do you see this being used? Is it for tests, or as part of an application?

Reply:
It is used to gets args for new signature of setcookie() call in ResponseEmitter.